pages/plugins/writing.md.erb

```diff
@@ -94,7 +97,20 @@ You can run the plugin linter with the following Docker command:
 docker run -it --rm -v "$PWD:/plugin:ro" buildkite/plugin-linter --id a-github-user/file-counter
 ```
 
-To make it easier to run this command, add it to the `docker-compose.yml` file:
+### Plugin linter plugin
```

Comment:
I don't think we need the repetition since it's clear from the plugin name.

```suggestion
### Plugin linter
```

Reply:
Not really... this is a buildkite plugin that runs the _linter for buildkite plugins_ so the plugin is not a repetition but an actual description that is necessary to differentiate between the linter itself and the plugin that runs the linter

Reply:
Maybe a better wording would be:
```suggestion
### Buildkite Plugin to run the plugin linter
```